pygmo/docstrings.cpp

```diff
@@ -4008,4 +4011,33 @@ Get the name of the NLopt solver used during construction.
 )";
 }
 
+std::string nlopt_local_optimizer_docstring()
+{
+    return R"(Local optimizer.
+
+Some NLopt algorithms rely on other NLopt algorithms as local/subsidiary optimizers.
+This property, of type :class:`~pygmo.core.nlopt`, allows to set such local optimizer.
```

Comment:
I have removed all the .core yesterday, it in the response to #34. For the fufture, do not use it. I will merge this and fix it on my coming PR